### PR TITLE
Some updates to the bootstrap task

### PR DESCRIPTION
**Pull request by oniofchaos**
I was going to create a script for setting up new user computers, but then I remembered about this. Making a few updates here instead.

@rreas @nvariz @shicholas please take a look. This would be good to iron out so we can use it once we get the new computers in.

### CODE REVIEW

File: rake_helpers/zsh.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def set_symlinks
       `ln -s #{zsh_config} "$HOME/.zshrc"`
       `ln -s #{zshenv_config} "$HOME/.zshenv"`
       # Homebrew and Chruby configuration
-      ln -s "$(brew --prefix)/Library/Contributions/brew_zsh_completion.zsh" /usr/local/share/zsh/site-functions
+      # ln -s "$(brew --prefix)/Library/Contributions/brew_zsh_completion.zsh" /usr/local/share/zsh/site-functions
```
Comment (oniofchaos) [inline rake_helpers/zsh.rb]: @shicholas can you take a look at this? Rake is throwing an error when it tries to load this file.

File: rake_helpers/code.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      f.lines[0].strip()
+    end
+
+    private
+
+    def setup_git_directories
+      `mkdir ~/Code`
+      `mkdir ~/Code/Work`
+      `mkdir ~/Code/Work/currica`
+      `cd ~/Code/Work/currica`
```
Comment (oniofchaos) [inline rake_helpers/code.rb]: This isn't working as expected. I need to fix it.

**Comment by shicholas** [comment]
Looks good, rvm is prob best for new computers than chruby. And I dig the folders for something to start with.

**Comment by oniofchaos** [comment]
@shicholas for your dotfiles configuration, I think we need to create the folders too. I'm testing it right now and I had trouble accessing .zshrc because ~/.dotfiles/zsh/zshrc didn't exist yet.

File: rake_helpers/code.rb (hunk 2 above)
Comment (shicholas) [inline rake_helpers/code.rb]: Does it not shell out properly? I'd try the ruby FileUtils module.

**Comment by shicholas** [comment]
Okay sounds good. I like using ``FileUtils.mkdir_p`` to keep things in the same ruby process

**Comment by shicholas** [comment]
Hmm actually I didn't document this well but fwiw I git cloned this repo to ``.~/.dotfiles`` that should solve the folder issue.

**Pull request merged**